resources/public/api/conf/1.0/application.raml

```diff
@@ -41,6 +42,7 @@ uses:
         businessId:
           description: 'An identifier for the business, unique to the customer.<p>Must conform to the regular expression `^X[A-Z0-9]{1}IS[0-9]{11}$`'
           example: XAIS12345678910
+      get: !include foreign_property_list.raml
```

Comment:
This is in the wrong place - needs to be just above the `/{submissionId}:` line.